src/casp/services/api/services/cell_quota_service.py

```diff
@@ -65,4 +67,25 @@ def get_quota_for_user(self, user: models.User) -> schemas.UserQuota:
             quota_reset_date=quota_reset_date,
             lifetime_quota=user.lifetime_cell_quota,
             remaining_lifetime_quota=remaining_lifetime_quota,
+            quota_increased=user.quota_increased,
         )
+
+    def increase_quota(self, admin_user: models.User, user_for_increase: models.User):
```

Comment:
Could you please add `-> None` in the method definition and remove :return: None from the docstring here as well?